recipe/build_libtorch.sh

```diff
@@ -84,7 +84,8 @@ if [[ "$CONDA_BUILD_CROSS_COMPILATION" == 1 ]]; then
     export COMPILER_WORKS_EXITCODE__TRYRUN_OUTPUT=""
 fi
 
-export MAX_JOBS=${CPU_COUNT}
+# reduce parallelism to avoid getting OOM-killed
+export MAX_JOBS=4
```

Comment:
would such a change be OK? It is useful to be able to able to run faster if you can.
```suggestion
if [[ "${CI}" == "github_actions" ]]; then
# h-vetinari/hmaarrfk -- May 2024
# reduce parallelism to avoid getting OOM-killed
# This has been tuned for our 
# runner: cirun-openstack-gpu-2xlarge
# that has 32GB of RAM and 8 CPU cores
  export MAX_JOBS=4
else
  export MAX_JOBS=${CPU_COUNT}
fi
```

i'm not sure if we can switch based on `${github_actions_labels}` but if we can that would be better

Reply:
> i'm not sure if we can switch based on `${github_actions_labels}` but if we can that would be better

That value would be populated according to CBC also in a local build, so that wouldn't achieve your goal AFAIU.